android/java/org/uproxy/tun2socks/DnsResolverService.java

```diff
@@ -1,288 +0,0 @@
-package org.uproxy.tun2socks;
```

Comment:
I will do that. There is also a branches that has an optimized version of the resolver.